Testscripts/Linux/ETHTOOL-OFFLOADING-SETTING.sh

```diff
@@ -133,7 +133,7 @@ if [ $? -ne 0 ]; then
 fi
 
 # Install iPerf3
-ssh -i "$HOME"/.ssh/"$SSH_PRIVATE_KEY" -o StrictHostKeyChecking=no "$remote_user"@"$VF_IP2" ". $HOME/utils.sh && update_repos && install_package iperf3"
+ssh -i "$HOME"/.ssh/"$SSH_PRIVATE_KEY" -o StrictHostKeyChecking=no "$remote_user"@"$VF_IP2" ". /home/${SUDO_USER}/utils.sh && update_repos && install_package iperf3"
```

Comment:
I don't understand this change here. What is your purpose?

Reply:
In most situation, this case is skipped, didn't run this logic at all. 
During testing, when it run install iperf3, the utils.sh file only exist under /home/$user folder.

Reply:
> I don't understand this change here. What is your purpose?

In most situation, this case is skipped, didn't run this logic at all. 
During testing, when it run install iperf3, the utils.sh file only exist under /home/$user folder.